db/buildData.js

```diff
@@ -0,0 +1,100 @@
+const path = require('path');
+const faker = require('faker');
```

Comment:
Looks like you ended up not using faker - probably a good idea, since it would slow down data gen a bit more for your purposes